generators/pkg/generator/generate_test.go

```diff
@@ -31,11 +31,7 @@ func TestGenerateTask(t *testing.T) {
 				"--destination=gcr.io/wlynch-test/kaniko-test",
 				"--verbosity=debug"},
 		}}}}
-	got, err := GenerateTask(spec)
-	if err != nil {
-		t.Fatalf("error from 'GenerateTask': %v", err)
-	}
-
+	got := GenerateTask(spec)
 	if diff := cmp.Diff(got, want); diff != "" {
```

Comment:
it should be `cmp.Diff(want,got)` for the printed diff to be of the form (-want/+got)

Reply:
sorry I' ll fix this